_posts/2017-09-14-survey-results.md

```diff
@@ -0,0 +1,218 @@
+
+We asked how easily Pa11y CI fits with our respondents workflows, with mixed results. 
+
+![Bar chart, scale of 1 (easier) to 5 (harder): How easy did you find it to integrate Pa11y CI with your workflow? 50% for 1; 25% for 3 and 4](asset/image/2017-survey/5-integrate.png)
+
+One respondent who found it more difficult to integrate with their workflow also reported to us that they work as a QA tester. They had a few suggestions that would help make Pa11y CI integrate more easily with their purposes:
+
+* HTML reporter output
+* errors in specified file 
+* acceptance of relative paths in sitemap.xml
+
+The boundaries between different IT job roles are becoming more blurred, and technology that used to be mostly used by developers is now reaching a much larger audience. In doing so, the technology also needs to be able to meet sometimes radically different use cases. 
+
+
+## What we plan to improve
+
+### Documentation, tutorials, and alternatives
+
+Since we launched the survey, we've made a number of improvements to the documentation for Pa11y and Pa11y Dashboard. This is an ongoing effort, but we feel that the documentation is in a pretty good place currently. We want to keep on improving that. 
+
+We're always on the lookout for high quality tutorials from users, as the Pa11y user community has shown itself to be excellent at writing them! A well-structured tutorial is often the best way for a new user to get up to speed, and we recognise how useful they are. We're collecting the most comprehensive tutorials we come across, and tweeting and linking out to them so that everyone can benefit.
+
+Not everyone who is interested in accessibility has the kind of skills needed to work with Pa11y. Pa11y Dashboard solves this to an extent, but it still needs to be set up and run on a server, and that's not feasible for everyone. For those users, a great option is [Koa11y](https://open-indy.github.io/Koa11y/), by Pa11y contributor [@thejaredwilcurt](https://github.com/TheJaredWilcurt). Koa11y wraps Pa11y in an installable desktop app. 
+
+
+### Code changes
+
+Since we launched the survey, we developed the [Actions](https://github.com/pa11y/pa11y#actions) interface, to provide an easier way for people to interact with their pages before running accessibility tests. On release, there was an immediate, dramatic decline in people needing support with interactions. The improvement in quality of life for our users since releasing Actions was so startling that we've decided to retire `beforeScript()` for Pa11y 5. We recognise that this means we'll need to add more Actions over time, and we're actively seeking suggestions. 
+
+In Pa11y 5, we'll be moving from PhantomJS to headless Chrome. The [Pa11y 5.x beta is already available](https://github.com/pa11y/pa11y/tree/5.x) for you to try out now. 
```

Comment:
Should this maybe link to the beta announcement rather than the tree? Once we merge and delete the 5.x branch this link will break